js/components/portal-dashboard/all-responses-popup/student-responses-popup.tsx

```diff
@@ -103,9 +109,23 @@ export class StudentResponsePopup extends React.PureComponent<IProps, IState> {
     );
   }
 
+  private handleChangeColorGroup = (studentId: string) => {
+    const { selectedStudents } = this.state;
+    const index = selectedStudents.findIndex((s: SelectedStudent) => s.id === studentId);
+    if (index >= 0) {
```

Comment:
As an aside I prefer this formulation ("index is valid") to the seemingly more common `index !== -1` which always reads to me as "result is not equal to the magic sentinel value for failure" which seems like an odd way to test for success.